### PR TITLE
[MRG] add test to confirm failure when summarizing on empty gather

### PR DESCRIPTION
**Pull request by hehouts**
so far:
added "summarize on empty gather.csv" test
will add a few more tests

adds to PR Ref #1543

### CODE REVIEW

**Comment by hehouts** [comment]
@bluegenes left a comment on the test im still working on

**Comment by hehouts** [comment]
@bluegenes I think this is ready to merge in now; the failed checks are in different tests

File: tests/test_tax.py [hunk 1]
```diff
@@ -745,19 +745,55 @@ def test_label_0(runtmp):
...
+def test_summarize_empty_gather_results(runtmp):
+    tax = utils.get_test_data('tax/test.taxonomy.csv')
+    
+    #creates empty gather result
+    g_csv = runtmp.output('g.csv')
+    with open(g_csv, "w") as fp:
+        fp.write("")
+    print("g_csv: ", g_csv)
+
+    #test below is not working properly: assert statement needs to be moved outside of the with block, but exc.value is empty and im not sure why
```
Comment (bluegenes) [inline tests/test_tax.py]:
```suggestion
```
this can go now, right? :)

File: tests/test_tax.py [hunk 2]
```diff
@@ -745,19 +745,55 @@ def test_label_0(runtmp):
...
+    outcsv = runtmp.output('out.csv')
+    g_csv = utils.get_test_data('tax/test1.gather.csv')
+    
+    with open(tax_empty, "w") as fp:
+        fp.write("")
+    print("t_csv: ", tax_empty)
+    import sys
+
+    with pytest.raises(ValueError) as exc:
+        runtmp.run_sourmash('tax', 'summarize', g_csv, '--taxonomy-csv', tax_empty, '--split-identifiers', '-o', outcsv)
```
Comment (bluegenes) [inline tests/test_tax.py]:
```suggestion
        runtmp.run_sourmash('tax', 'summarize', g_csv, '--taxonomy-csv', tax_empty)
```

File: tests/test_tax.py [hunk 3]
```diff
@@ -745,19 +745,55 @@ def test_label_0(runtmp):
...
+    print(f"----------------- [DEBUG]{exc.value}", file=sys.stderr)
+    if(str(exc.value) == "local variable 'n' referenced before assignment"):
```
Comment (bluegenes) [inline tests/test_tax.py]:
```suggestion
    assert f"No taxonomic assignments loaded from {tax_empty}" in str(exc.value
```

Comment (bluegenes) [reply tests/test_tax.py]: I think this is the relevant err to catch now!

File: tests/test_tax.py (hunk 1 above)
Comment (hehouts) [reply tests/test_tax.py]: oh oops ok, this is the wrong version. Im on the github struggle bus.

**Review by bluegenes — APPROVED** [review]
thanks @hehouts! I'll figure out what error I want to use for that 2nd one!
